src/components/leaf/MainCardInput.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { useState } from 'react';
+import MainCardInput from './MainCardInput';
+
+const meta: Meta<typeof MainCardInput> = {
+  component: MainCardInput,
+  title: 'Leaf/Input',
```

Comment:
すみません、今気づきました！
```suggestion
  title: 'Leaf/MainCardInput',
```
OKなら　`Commit suggestion`ボタンでマージをお願いします！
（その後ローカル作業する場合はpullも忘れずに！）